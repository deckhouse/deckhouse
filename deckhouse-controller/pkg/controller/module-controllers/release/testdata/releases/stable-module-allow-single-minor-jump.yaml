apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: test-module
  namespace: default
spec:
  version: "1.0.0"
  source: test-source
  releaseChannel: "Stable"
status:
  phase: Ready
  conditions:
  - type: IsReady
    status: "True"
  - type: EnabledByModuleManager
    status: "True"
  - type: EnabledByModuleConfig
    status: "True"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  name: test-source
  namespace: default
spec:
  registry:
    repo: registry.deckhouse.io/deckhouse/ce/modules
    scheme: HTTPS
status:
  phase: Active
  modules:
  - name: test-module
    version: "1.1.0"
    checksum: sha256:test-checksum
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: test-module-v1.0.0
  namespace: default
  labels:
    module: test-module
spec:
  moduleName: test-module
  version: "1.0.0"
  weight: 900
status:
  phase: Deployed
  message: ""
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: test-module-v1.1.0
  namespace: default
  labels:
    module: test-module
spec:
  moduleName: test-module
  version: "1.1.0"
  weight: 900
status:
  phase: Pending
  message: ""
