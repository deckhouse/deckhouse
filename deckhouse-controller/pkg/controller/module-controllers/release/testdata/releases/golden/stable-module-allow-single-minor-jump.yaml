---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  creationTimestamp: null
  name: test-source
  resourceVersion: "999"
spec:
  registry:
    ca: ""
    dockerCfg: ""
    repo: registry.deckhouse.io/deckhouse/ce/modules
    scheme: HTTPS
status:
  message: ""
  modules:
  - checksum: sha256:test-checksum
    name: test-module
    version: 1.1.0
  modulesCount: 0
  phase: Active
  syncTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: test-module
    source: test-source
  name: test-module-v1.0.0
  resourceVersion: "1000"
spec:
  moduleName: test-module
  version: 1.0.0
  weight: 900
status:
  approved: false
  message: ""
  phase: Superseded
  pullDuration: 0s
  size: 0
  transitionTime: "2019-10-17T15:33:00Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    modules.deckhouse.io/isUpdating: "true"
    modules.deckhouse.io/notified: "false"
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/exist-on-fs
  labels:
    module: test-module
    modules.deckhouse.io/update-policy: ""
    source: test-source
    status: deployed
  name: test-module-v1.1.0
  resourceVersion: "1004"
spec:
  moduleName: test-module
  version: 1.1.0
  weight: 900
status:
  approved: false
  message: ""
  phase: Deployed
  pullDuration: 9m15s
  size: 0
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  creationTimestamp: null
  name: test-module
  resourceVersion: "999"
properties: {}
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: null
    status: "True"
    type: IsReady
  - lastProbeTime: null
    lastTransitionTime: null
    status: "True"
    type: EnabledByModuleManager
  - lastProbeTime: null
    lastTransitionTime: null
    status: "True"
    type: EnabledByModuleConfig
  phase: Ready
